### DISCUSSION TITLE
Proposal to Prioritize Studio Tracks over Sampler Contributions/Live Recordings in Top Song Generation

### DISCUSSION THREAD

**Discussion by Lokke** [navidrome/navidrome#2890] [Ideas]
I propose lowering the priority of sampler contributions in generating the top songs and consistently opting for only the song with the lowest year indication.

Frequently, live recordings or sampler contributions are featured, which I personally find disruptive.

**Comment by metalheim** [comment]
This is related (kinda similar) to https://github.com/navidrome/navidrome/issues/1701 that i opened because i was facing the same issue.

Navidrome already de-emphazises Compilations as of https://github.com/navidrome/navidrome/pull/2532
so if you tag the samplers as "compilation", you'll solve part of your issue.

There also was an issue with getTopSongs described here: https://github.com/navidrome/navidrome/issues/2670
 that got fixed in december. That Issue also provides a good lot of information on how the selection in the findMatchingTrack function (that is used in getTopSong) works.

Live recordings (from the same year as original) are a bit harder, as navidrome would currently have no clue what a live version is and what isn't.
Since i don't have a lot of live versions - how often does it happen that there's a live version from the same year as the original?
